Website/src/rendering/src/models/cdp/AddToCartPayload.ts

```diff
@@ -0,0 +1,22 @@
+export type AddToCartPayload = {
+  channel: string;
```

Comment:
channel, type, language, currency, page, pos, and browser_id are all managed already in BoxeverService. Maybe they could be removed from this and the other model?